.github/workflows/integration_tests.yml

```diff
@@ -102,6 +102,13 @@ jobs:
     - uses: ./.github/set_up_cromwell_action #This github action will set up git-secrets, caching, java, and sbt.
       with:
         cromwell_repo_token: ${{ secrets.BROADBOT_GITHUB_TOKEN }}
+    # Activate SSH and idle for 30 minutes
+#    - name: Setup tmate session
+#      uses: mxschmitt/action-tmate@v3
+#      timeout-minutes: 30
+#      with:
+#        limit-access-to-actor: true
+#        detached: true
     #This script bascially just looks up another script to run, assuming that the other script's filename is:
```

Comment:
I assume this is left in here commented-out intentionally and I support it 👍

Reply:
Yes, that is correct!